homeassistant/components/switch/fritzbox_wifi.py

```diff
@@ -0,0 +1,100 @@
+"""
+Fritzbox (Guest) WiFi Switch.
+
+Support for switching Fritzbox (Guest) Wifi on and off.
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/switch.fritzbox_wifi/
+"""
+# pylint: disable=import-error
+
+import logging
+import voluptuous as vol
+
+from homeassistant.components.switch import (SwitchDevice, PLATFORM_SCHEMA)
+from homeassistant.const import (
+    CONF_HOST, CONF_PORT, CONF_NAME, CONF_USERNAME, CONF_PASSWORD)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['fritzconnection==0.6.5']
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = 'Guest Wifi'
+DEFAULT_HOST = '169.254.1.1'
+DEFAULT_PORT = 49000
+DEFAULT_INTERFACE = 3
+
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_HOST, default=DEFAULT_HOST): cv.string,
+    vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.positive_int,
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_USERNAME): cv.string,
+    vol.Required(CONF_PASSWORD): cv.string,
+    vol.Optional('interface', DEFAULT_INTERFACE): cv.positive_int
+})
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the Fritzbox WiFi switch platform."""
+    import fritzconnection as fc
+    host = config.get(CONF_HOST)
```

Comment:
Use `dict[key]` for required config keys and keys with default config schema values.